projects/bifrost/index.js

```diff
@@ -0,0 +1,109 @@
+const { ApiPromise, WsProvider } = require("@polkadot/api");
+const { options } = require("@bifrost-finance/api");
```

Comment:
can `@acala-network/api`  be used here and avoid an extra npm dependency?

Reply:
Hello , we can't use @acala-network/api , we have a lot of different types.

Reply:
Hello I removed the types , I think we don't need it.